packages/next/src/server/lib/start-server.ts

```diff
@@ -312,11 +374,27 @@ export async function startServer({
         proxyServer.on('proxyRes', (proxyRes) => {
           res.on('close', () => proxyRes.destroy())
         })
-
+        req.on('error', (_err) => {
+          // TODO: log socket errors?
+        })
+        res.on('error', (_err) => {
+          // TODO: log socket errors?
+        })
         proxyServer.web(req, res)
       }
       upgradeHandler = async (req, socket, head) => {
+        req.on('error', (_err) => {
+          // TODO: log socket errors?
+          // console.log(_err)
+        })
+        socket.on('error', (_err) => {
+          // TODO: log socket errors?
+          // console.log(_err)
+        })
```

Comment:
Let's comment out the socket.on? Or if it's needed add a comment explaining what it does.